man/systemd.service.xml

```diff
@@ -1123,8 +1123,14 @@
       <varlistentry>
         <term><varname>OOMPolicy=</varname></term>
 
-        <listitem><para>Configure the Out-Of-Memory (OOM) killer policy. On Linux, when memory becomes scarce
-        the kernel might decide to kill a running process in order to free up memory and reduce memory
+        <listitem><para>Configure the out-of-memory (OOM) kernel killer policy. Note that the userspace OOM
+        killer
+        <citerefentry><refentrytitle>systemd-oomd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        is a more flexible solution that is allows preventing out-of-memory situations for the userspace, not
```

Comment:
/that is allows/that allows/

Reply:
I reworded that as "that aims to prevent". I don't want to say "prevents", because it can't cover all cases.